addons/addon-base-post-deployment/packages/base-post-deployment/package.json

```diff
@@ -9,7 +9,7 @@
     "@amzn/base-api-services": "workspace:*",
     "@amzn/base-services": "workspace:*",
     "@amzn/base-services-container": "workspace:*",
-    "aws-sdk": "^2.1000.0",
```

Comment:
it looks like a downgrade `aws-sdk` and the latest one is `2.1233.0`, is it expected?

Reply:
Upgrade all to ^2.1233.0 version